### PR TITLE
Add analyser_merge script for public transport data

### PR DESCRIPTION
**Pull request by Futur3r**
Add an analyser_merge script for the GTFS data of the SIBRA public transport network based in Annecy

### CODE REVIEW

**Comment by frodrigo** [comment]
Please can you also add it to the osmose_config.py

File: osmose_config.py [hunk 1]
```diff
@@ -475,8 +475,10 @@ class gen(default_country):
     'merge_street_number_lyon',
 ])
 france_departement("rhone_alpes/savoie", 7425, "FR-73")
-france_departement("rhone_alpes/haute_savoie", 7407, "FR-74")
-
+france_departement("rhone_alpes/haute_savoie", 7407, "FR-74", include=[
+    # Annecy
+    'merge_public_transport_FR_sibra'
+])
```
Comment (frodrigo) [inline osmose_config.py:481]:
```suggestion
])

```

Comment (frodrigo) [inline osmose_config.py:481]:
```suggestion
])

```

File: analysers/analyser_merge_public_transport_FR_sibra.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2017                                      ##
```
Comment (frodrigo) [inline analysers/analyser_merge_public_transport_FR_sibra.py:6]: Please, can you set you name here.

File: analysers/analyser_merge_public_transport_FR_sibra.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+from .Analyser_Merge import Analyser_Merge, Source, GTFS, Load, Conflate, Select, Mapping
+
+
+class Analyser_Merge_Public_Transport_FR_sibra(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        place = "SIBRA"
+        self.def_class_missing_official(item = 8040, id = 91, level = 3, tags = ['merge', 'public transport'],
+            title = T_('{0} stop not integrated', place))
+        self.def_class_possible_merge(item = 8041, id = 93, level = 3, tags = ['merge', 'public transport'],
```
Comment (frodrigo) [inline analysers/analyser_merge_public_transport_FR_sibra.py:33]:
```suggestion
        self.def_class_missing_official(item = 8040, id = 101, level = 3, tags = ['merge', 'public transport'],
            title = T_('{0} stop not integrated', place))
        self.def_class_possible_merge(item = 8041, id = 103, level = 3, tags = ['merge', 'public transport'],
```

File: osmose_config.py [hunk 4]
```diff
@@ -475,7 +475,10 @@ class gen(default_country):
     'merge_street_number_lyon',
 ])
 france_departement("rhone_alpes/savoie", 7425, "FR-73")
-france_departement("rhone_alpes/haute_savoie", 7407, "FR-74")
+france_departement("rhone_alpes/haute_savoie", 7407, "FR-74", include=[
+    # Annecy
+    'merge_public_transport_FR_sibra'
```
Comment (frodrigo) [inline osmose_config.py:480]:
```suggestion
    'merge_public_transport_FR_sibra',
```

File: analysers/analyser_merge_public_transport_FR_sibra.py (hunk 2 above)
Comment (Futur3r) [reply analysers/analyser_merge_public_transport_FR_sibra.py]: Done

**Comment by frodrigo** [comment]
Thank you. Merged, but nor deployed yet.

**Comment by frodrigo** [comment]
Deployed
http://osmose.openstreetmap.fr/fr/errors/?item=8040,8041&class=101,103

**Pull request merged**